apps/service-portal/src/assets/images/404.svg

```diff
@@ -0,0 +1,25 @@
+<svg width="380" height="380" viewBox="0 0 380 380" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
Could probably be shrinked via something like https://jakearchibald.github.io/svgomg/